src/tabs/console/ConsoleTabView.tsx

```diff
@@ -83,4 +89,19 @@ export class ConsoleTabView extends TabView<ConsoleTabState> {
     return "rgb(" + r.toString() + "," + g.toString() + "," + b.toString() + ")";
   }
 
+  private autoComplete(command: string) {
+    if (this.state.commands.length === 0) {
+      this.getCommands();
+    }
+    ConsoleAutocomplete.setCommandList(this.state.commands);
+    this.props.model.update({commandToSend: ConsoleAutocomplete.complete(command)});
+  }
+
+  private getCommands(): any {
+    this.props.model.requestResource({
```

Comment:
Since the Model is subscribed to the resource I think the data you need is already in `this.state.commands`. Are you doing the request to always have an updated version? I don't think it's necessary because if I recall correctly the set of commands is always the same

Reply:
It looks like it does work without this, I will remove it.